strategies/candlesticks/white_shark.js

```diff
@@ -38,19 +38,18 @@ class WhiteShark {
 
   // format of the single candle: [ time, low, high, open, close, volume ]
   candles(values) {
-    console.log(' Received candles ');
-    values.array.forEach((el) => {
-      console.log(`Ts:${moment.unix(el[0]).format('DD/MM/YYYY@HH:mm:00')}, 
-                   lo:${el[1]}, hi:${el[2]}, op:${el[4]}, close:${el[5]}, vol:${el[6]}`);
+    values.forEach((el) => {
+      //this.logger.debug(`Ts:${moment.unix(el[0]).format('DD/MM/YYYY@HH:mm:00')}, 
+      //             lo:${el[1]}, hi:${el[2]}, op:${el[4]}, close:${el[5]}, vol:${el[6]}`);
     });
     const lastCandle = values[0];
     const secondLastCandle = values[1];
     if (secondLastCandle[3] < secondLastCandle[4] // second last is 'red' (e.g. 'open' is lower than 'close')
         && lastCandle[3] > lastCandle[4] // most recent is 'green' (e.g. 'open' is higher than 'close')
         && lastCandle[1] - secondLastCandle[2] > 0 // 'low' of the last higher than the 'high' of the second last
     ) {
-      console.log(' Bullish Pattern detected');
-      console.log(` Liquidity:${this.moneyAmount}`);
+      //this.logger.debug(' Bullish Pattern detected');
```

Comment:
Codacy has a fix for the issue: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
      // this.logger.debug(' Bullish Pattern detected');
```